### ISSUE TITLE
Pressing "Enable Google Drive" checkbox option leaves it unchecked

### ISSUE THREAD

**Issue by rivaldi8** [codinguser/gnucash-android#564]
Reported through Google+:
> When I tick the "enable Google drive" tick box under "backup settings", a little pop-up notification says "connected to Google drive" but the box isn't ticked. Then I close the settings and reopen the settings and it is already ticked.

**Comment by linuxluser** [comment]
When you first check the box, you get a notification: "Connected to Google Drive" but the checkmark does not change. If you press the back button and then touch the "Restore Backup ..." option again, you can visibly seen the checkmark. See ![screenshot](https://cloud.githubusercontent.com/assets/41452/18979669/f38a57bc-8681-11e6-95e4-cef748cc1943.png).

**Comment by codinguser** [comment]
Yes, I can confirm this. But it was/is very low priority bug for me.
